macros/src/types/newtype.rs

```diff
@@ -29,6 +29,7 @@ pub(crate) fn newtype(
         skip,
         optional,
         flatten,
+        docs: _, // TODO: Not sure what to do here with docs yet
```

Comment:
I actually think discarding is the right thing to do, documenting the field of a newtype seems pointless to me, just document the type itself